src/processpriv.cpp

```diff
@@ -66,17 +67,20 @@ ProcessPriv::ProcessPriv(const char* command, const QoreListNode* arguments, con
     ResolvedCallReferenceNode* on_exec_error = optsExecutor("on_exec_error", opts, xsink);
 
     // parse options
-    bp::environment e = optsEnv(opts, xsink);
+    bp::environment env = optsEnv(opts, xsink);
     boost::filesystem::path p = optsPath(command, opts, xsink);
-    const char* cwd = optsCwd(opts, xsink);
+    std::string cwd = std::move(optsCwd(opts, xsink));
```

Comment:
no need for move for returned values, either copy elision takes place or implicit move

"The part of the Standard blessing the RVO goes on to say that if the conditions for the RVO
are met, but compilers choose not to perform copy elision, the object being returned
must be treated as an rvalue."